### PR TITLE
fix invocation with docker

### PR DESCRIPTION
**Pull request by bard**
Fixes https://github.com/bats-core/bats-core/issues/437.

### CODE REVIEW

File: docs/source/docker-usage.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ This runs the `test/` directory from the bats-core repository inside the bats Do
 For test suites that are intended to run in isolation from the project (i.e. the tests do not depend on project files outside of the test directory), you can mount the test directory by itself and execute the tests like so:
 
 ```bash
-$ docker run -it -v "$PWD/test:/test" bats/bats:latest /test
+$ docker run -it -v "$PWD/test:/test" bats/bats:latest /code/test
```
Comment (martin-schulze-vireso) [inline docs/source/docker-usage.md:35]: I think that is missing /code in the Volume mapping.

**Review by martin-schulze-vireso — CHANGES_REQUESTED** [review]
You mentioned a second documentation location that die not work.

File: docs/source/docker-usage.md (hunk 1 above)
Comment (bard) [reply docs/source/docker-usage.md]: Quite worrying when I'm unable to copy/paste from my own report...

**Comment by bard** [comment]
> You mentioned a second documentation location that die not work.

Yes, but I couldn't find it in current docs, only in a PR: https://github.com/bats-core/bats-core/pull/274

**Comment by martin-schulze-vireso** [comment]
> Yes, but I couldn't find it in current docs

It's here: https://github.com/bats-core/bats-core/blame/master/docs/source/installation.rst#L136

**Comment by dilawar** [comment]
I ran into this problem today. I was trying out bats. 

```
[admin@ip-172-26-13-230 postgrest (issue_4)]$ make test
./bin/bats.sh tests
+ docker run -it --volume /home/admin/Work/canary/api/postgrest/:/opt/bats --workdir /opt/bats bats/bats:latest tests
bash: /usr/sbin/bats: No such file or directory
make: *** [Makefile:16: test] Error 127
```

**Review by martin-schulze-vireso — APPROVED** [review]

**Pull request merged**